.github/workflows/test.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Tests
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        go: [ '1.11.x', '1.12.x', '1.13.x', '1.14.x', '1.15.x', '1.16.x' ]
+    steps:
+      - name: Checkout smtpapi-go
+        uses: actions/checkout@v2
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Set GOPATH
+        run: echo "GOPATH=$HOME" >> $GITHUB_ENV
+      - name: Set GOBIN
```

Comment:
Why set `GOBIN`?

Reply:
In sg-go, not setting `GOBIN` yielded [a bunch of errors](https://github.com/sendgrid/sendgrid-go/runs/4358686052?check_suite_focus=true) with `go get` (whereas [setting it](https://github.com/sendgrid/sendgrid-go/runs/4358732353?check_suite_focus=true) fixed the issue).